src/main/java/by/itech/lab/supplier/service/impl/WarehouseServiceImpl.java

```diff
@@ -90,33 +90,40 @@ public WarehouseDto findById(final Long warehouseId) {
         return warehouseRepository.findAllByRetailerId(retailerId, pageable).map(warehouseMapper::map);
     }
 
+    @Override
+    public Boolean isContainsIdentifier(final String identifier) {
+        Long id = warehouseRepository.findByIdentifier(identifier);
```

Comment:
findByIdentifier should return optional. If it's not for now return may be the single line in this method, don't need to create id variable.